webpack/assets/javascripts/react_app/components/GraphQLTest/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import gql from 'graphql-tag';
+
+import { withGraphQLData } from '../../common/graphql';
+
+const GET_MODELS = gql`
```

Comment:
We might consider using [webpack loader](https://medium.com/modus-create-front-end-development/load-your-graphql-documents-with-webpack-apollo-c1e166404cba) in the future, probably ok to keep the queries together with components for now.